engines/bladerunner/font.cpp

```diff
@@ -68,6 +110,27 @@ bool Font::open(const Common::String &fileName, int screenWidth, int screenHeigh
 		_characters[i].width = stream->readUint32LE();
 		_characters[i].height = stream->readUint32LE();
 		_characters[i].dataOffset = stream->readUint32LE();
+#if SUBTITLES_SUPPORT
+#if !SUBTITLES_EXTERNAL_FONT
+		// special explicit alignment fixes for TAHOMA18 (INTERNAL) font
+		if (fileName == "TAHOMA18.FON") {
+			// fix P -> i = 81 (ascii code 80  + 1)
```

Comment:
A switch statement would make this a bit more readable in my opinion.

Reply:
I could try that. The current if clauses and line separation with comments reflect my groupings of characters that had similar behavior as far as their placement/kerning go.